PubCloud enhancements (#272)
uyuni-project/uyuni-docs: modules/quickstart-public-cloud/pages/installation.adoc

[IMPORTANT]
====

> Before bootstrapping a client, all the selected channels for that product need to be synchrozined.

Reviewer: ```suggestion
Before bootstrapping a client, make sure all the selected channels for that product are synchronized.
```
